pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -255,7 +255,11 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
             # Don't type pinhole, dark, or bias frames
             return np.zeros(len(fitstbl), dtype=bool)
         if ftype in ['arc', 'tilt']:
```

Comment:
Do we need self.dispname if we have the fitstable?

Reply:
self.dispname is always None here. I think check_frame_type runs before you init self.dispname with a science frame.

Reply:
Yeah that is right. This is fine.